scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -130,8 +133,15 @@ if [ "$silent" == "true" ]; then
 else
     kubectl ws "${kubectl_flags[@]}" "$imw"
 fi
+spaceID=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}")
 
-mbwsname=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
+if [ "$central" == "true" ]; then
```

Comment:
In what call stack is this ever NOT wanted?

Reply:
Why is there a `--central` flag?

Reply:
Was thinking there could be a need to search for a synctarget in IS. Removed.

Reply:
Done.